src/app/app.component.html

```diff
@@ -17,11 +17,6 @@
   <mat-grid-tile [colspan]="1" [rowspan]="3">
     <img src="assets/logo-lightning-talk.png" alt="Logo da Lightning Talk">
   </mat-grid-tile>
-  <mat-grid-tile [colspan]="1" [rowspan]="1">
```

Comment:
@mkacunha what happens when user no have active session?

Reply:
Probably the MsalInterceptor to force a new login when a request by way HttpClient return status 401 or 403. I will do a test.

Reply:
@altitdb The MsalInterceptor to force a new login when a request by way HttpClient return status 401 or 403